packages/ffe-icons-react/.babelrc

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+        "development": {
+            "presets": [
+                ["env", {
+                    "targets": { "browsers": "ie 11, last 2 versions" }
+                }],
+                "stage-3",
+                "react"
+            ]
+        },
+        "es": {
+            "presets": [
+                ["env", {
+                    "modules": false,
+                    "targets": { "browsers": "ie 11, last 2 versions" }
+                }],
+                "stage-3",
+                "react"
+            ]
+        }
+    }
+}
```

Comment:
This can be simplified greatly by upgrading to Babel 7. I'll attack that task tomorrow.

Reply:
Yes! Do it! :)
Beware you might need a dependency `"babel-core": "7.0.0-bridge.0"`, unless jest has released a new version that doesn't depend on babel 6.